doc/tensor.md

```diff
@@ -1327,36 +1331,64 @@ x:indexFill(2,torch.LongTensor{4,2},-10)
 ```
 
 <a name="torch.Tensor.gather"/>
-### [Tensor] gather(src, dim, index) ###
+### [Tensor] torch.gather(src, dim, index) ###
```

Comment:
I've changed it to `[Tensor] gather(dim, index)` which makes it consistent with the rest of the docs, and re-worded slightly to introduce `src` when it is needed.